convert/src/vertex_to_bd/labelstudio_format/labelstudio_format.py

```diff
@@ -0,0 +1,83 @@
+from pathlib import Path
+from typing import List
+
+from ..models.bd_annotation_model import BadgerdocAnnotation
+from ..models.bd_tokens_model import BadgerdocToken, Page
+from ..models.vertex_models import (
+    Annotation,
+    Data,
+    LabelStudioModel,
+    ModelItem,
+    ResultItem,
+    Value,
+)
+
+
+class LabelStudioFormat:
+    # TODO: support more than 1 page
+    def __init__(self) -> None:
+        self.labelstudio_data = LabelStudioModel()
+
+    def from_badgerdoc(
+        self,
+        badgerdoc_tokens: Page,
+        badgerdoc_annotations: BadgerdocAnnotation,
+    ):
+        text = "".join([obj.text for obj in badgerdoc_tokens.objs])
+        objs = self.convert_annotation_from_bd(
+            badgerdoc_annotations, badgerdoc_tokens.objs
+        )
+        relations = self.convert_relation_from_bd(
+            badgerdoc_annotations, badgerdoc_tokens.objs
+        )
+        item = ModelItem(
+            annotations=[Annotation(result=objs + relations)],
+            predictions=[],
+            data=Data(text=text),
+        )
+        self.labelstudio_data.__root__.append(item)
```

Comment:
Open question as above: should we really modify this object? If so, rename the method